Makefile

```diff
@@ -448,14 +448,14 @@ staging-manifests:
 ## --------------------------------------
 ##@ Release
 ## --------------------------------------
-
+RELEASE_TAG ?= $(shell git describe --abbrev=0 2>/dev/null)
```

Comment:
This is already defined here: https://github.com/kubernetes-sigs/cluster-api-provider-openstack/blob/b9fa78bbfef0c2a64cdba8af3eec65d0926178ff/Makefile#L97
It is ok to move but we should not keep both.

Reply:
Ok I kept the original one